dsl/src/main/java/tech/pegasys/peeps/network/Network.java

```diff
@@ -173,25 +189,30 @@ public Besu addNode(
         privacyManager);
 
     return addNode(
-        new BesuConfigurationBuilder()
+        new Web3ProviderConfigurationBuilder()
             .withIdentity(identity)
             .withNodeKey(ethereumIdentiity)
             .withPrivacyUrl(privacyManagers.get(privacyManager))
-            .withPrivacyManagerPublicKey(privacyAddressResource.get()));
+            .withPrivacyManagerPublicKey(privacyAddressResource.get()),
+        Web3ProviderType.BESU);
   }
 
-  private Besu addNode(final BesuConfigurationBuilder config) {
-    final Besu besu =
-        new Besu(
-            config
-                .withVertx(vertx)
-                .withContainerNetwork(subnet.network())
-                .withIpAddress(subnet.getAddressAndIncrement())
-                .withGenesisFile(genesisFile)
-                .withBootnodeEnodeAddress(bootnodeEnodeAddresses())
-                .build());
+  private Web3Provider addNode(
+      final Web3ProviderConfigurationBuilder config, final Web3ProviderType providerType) {
+    final Web3Provider web3Provider;
+    config
+        .withVertx(vertx)
```

Comment:
do we need to set this if we are building `GoQuorum` ? or it will simply be ignored by GoQorum constructor?

Reply:
The Vertx is how the test frameworks talks to the node, i.e. it doesn't get passed _into_ the node.